benchexec/tablegenerator/react-table/src/workers/scripts/stats.worker.js

```diff
@@ -0,0 +1,392 @@
+// This file is part of BenchExec, a framework for reliable benchmarking:
+// https://github.com/sosy-lab/benchexec
+//
+// SPDX-FileCopyrightText: 2019-2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// COPY OF utils.js, as imports will not work here
```

Comment:
Hm, this seems error prone. At the very least, there must be a similar comment at the original location. But can we do better? For example, move common code to a separate file and include both in the worker URI (the code for this already seems to support an arbitrary number of files)?

Reply:
The idea of moving the code into the common section of the dataURIs (at least thats how i interpret the comment) is an interesting idea. 
I am however not sure if this will work, as the dataURI should be loaded as a worker without any context/closures as it is basically just text. 

I see two options: Either we apply the same comment in the areas that were copied over into worker files or we extend the dataURI creation script to inject worker-global code into the dataURIs while building.

One issue that i see with injecting worker-global code into the dataURIs is that we might clash with functions/variables defined in the worker because we are suddenly shadowing objectsthat were not previously present. We could try to mitigate this issue with some clever usage of JSDoc but, depending on the IDE and the used extensions, this might still result in problems. 

What are your thoughts on that? 
For now i will simply add another comment in the utils script

Reply:
I understand. I might have suggested simply concatenating global stuff with the worker code, but silent shadowing seems like a bad thing to debug. So I don't have a better solution than adding comments.